docs/content/preview/benchmark/sysbench-ysql.md

```diff
@@ -39,55 +69,63 @@ Make sure you have the [YSQL shell](../../api/ysqlsh/) `ysqlsh` exported to the
 $ export PATH=$PATH:/path/to/ysqlsh
 ```
 
```

Comment:
Do "This installs the sysbench utility in `/usr/local/bin`." and the ysqlsh PATH instructions not apply to all 3 install types ie put these after the last tab